export const AccountErrors = {
    DuplicatedEmail: {
        errorMessage: 'Email is already being used.',
        reason: 'Email is already taken by another user.',
    },
    UknownError: {
        errorMessage: 'Something weird happenend, check logs for more detail.',
        reason: 'Unknown internal server error.'
    }
};
